src/PrestaShopBundle/Form/Admin/Improve/Design/Pages/CmsPageType.php

```diff
@@ -50,6 +49,11 @@
  */
 class CmsPageType extends TranslatorAwareType
 {
+    private const TITLE_MAX_CHARS = 255;
```

Comment:
I see no reason to put this const in private

Reply:
The class is not final, it can be extend, if you put it in private, no one can use it

Reply:
Good point. Made all consts like this in both page and page category public.